ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKStreamTest.java

```diff
@@ -73,9 +73,10 @@ public void init() {
     ksqlStream = (KsqlStream) metaStore.getSource("TEST1");
     StreamsBuilder builder = new StreamsBuilder();
     kStream = builder.stream(ksqlStream.getKsqlTopic().getKafkaTopicName(),
-        Consumed.with(Serdes.String(), ksqlStream.getKsqlTopic()
-            .getKsqlTopicSerDe().getGenericRowSerde(null, new KsqlConfig(Collections.emptyMap())
-                , false, new MockSchemaRegistryClient())));
+                             Consumed.with(Serdes.String(), ksqlStream.getKsqlTopic()
```

Comment:
same as above. i.e., formatting doesn't look like it is inline with the template

Reply:
Reformatted with our code style. Some spaces have remained.